includes/admin/reporting/graphing.php

```diff
@@ -667,11 +667,19 @@ function edd_reports_graph_controls() {
 						$to   = empty( $dates_values['to'] )   ? '' : $dates_values['to'];
 						?>
 						<span class="edd-search-date">
-						<span><?php _ex( 'From', 'date filter', 'easy-digital-downloads' ); ?></span>
-							<?php echo EDD()->html->date_field( array( 'name' => 'filter_from', 'placeholder' => __( 'mm/dd/yyyy', 'easy-digital-downloads' ), 'value' => $from ) ); ?>
+							<span><?php _ex( 'From', 'date filter', 'easy-digital-downloads' ); ?></span>
+							<?php echo EDD()->html->date_field( array(
+								'name'        => 'filter_from',
+								'placeholder' => 'yyyy-mm-dd',
+								'value'       => $from
+							) ); ?>
 
 							<span><?php _ex( 'To', 'date filter', 'easy-digital-downloads' ); ?></span>
-							<?php echo EDD()->html->date_field( array( 'name' => 'filter_to', 'placeholder' => __( 'mm/dd/yyyy', 'easy-digital-downloads' ), 'value' => $to ) ); ?>
+							<?php echo EDD()->html->date_field( array(
+								'name'        => 'filter_to',
+								'placeholder' => 'yyyy-mm-dd',
```

Comment:
Same question on the entered format. I think it was previously mm/dd/yyyy because that's "more familiar".

Reply:
The jQuery UI Datepicker got changed to `yy-mm-dd` so it no longer matches the format or pattern.

I did forget to use the new function here though, so I’ll note that.

Reply:
Updated to use `edd_get_date_picker_format()` instead of `yyyy-mm-dd` or any other hard-coded string.